tensorflow/contrib/tensorrt/README.md

```diff
@@ -0,0 +1,42 @@
+Using TensorRT in TensorFlow
```

Comment:
This PR contains much more than the title "introducing TensorRT operator". I'd like to see a more detailed description listing specific changes. For example, 

- Add a TRTEngineOp that encapsulates a TensorRT executable. 
- Add CreateInferenceGraph to contract a TensorRT-compilable subgraph to a TRTEngineOp. 
- Add configurations that blah blah.

Reply:
Updated the PR message. Is this better?